src/PJ_aitoff.c

```diff
@@ -49,12 +49,6 @@ PROJ_HEAD(aitoff, "Aitoff") "\n\tMisc Sph";
 PROJ_HEAD(wintri, "Winkel Tripel") "\n\tMisc Sph\n\tlat_1";
 
 
-
-#if 0
```

Comment:
Not a problem removing this. It does, however, reflect the general issue that we are still working on a code base that is highly inconsistent and architecturally challenged: There is a multitude of these cases where a focused development effort leads to the discovery of quirks and oddities that may either be solved drive-by style, or described in the issue tracker and deferred to a separate PR.

I tend to take the drive-by approach, when the effort of writing up the issue-description, as in this case, *significantly* outweighs the effort needed to *just solve it*.

I discovered this case of conditional compilation when removing 3 cases of temporarily introduced `#if 0`/`#endif` pairs during debugging. And since the `pj_aitoff.c` case was the *only* other case of conditional compilation left, I decided to remove it, while at it.

In another case (the latitude-overshoot `EPS`, mentioned elsewhere), I just introduced the needed constant, and used it where it made sense for this PR. Later, due to a refactor, that constant ended up being needed in just one file, rather than originally 3, within the scope of the PR. Outside of the scope of the PR there are plenty of non-trivial cases where it should be introduced, but that is deferred to a separate PR.

But what is inside, and what is outside of scope is a matter of judgement. An unlikely, but different judgement, could have been that introducing `PJ_LAT_EPS`, to clarify some code in scope, directly pulls all other `EPS` referring files into the scope of this PR.

This is not to praise a cowboy coding style, it is more a reflection that when travelling the trajectories of the PROJ code base, we are still very much in Cowboy-land, and while we should strive to civilize this wild west, it is still often best done by repairing what is seen to need repair. I think the rule-of-thumb mentioned above (“is writing an issue description a *much* bigger effort than just doing it”) is a useful discriminator.

That said, I’m really looking forward to working in a code base that is well enough structured, well enough documented, and well enough understood, to skip the drive-by fixes entirely - but the trip from Cowboy-land to Nirvana is still long (and probably even never-ending).

Reply:
I don't mind fixing it as part of the pull request, that'll be transparent in the history anyway. I suggested you do this outside this PR since you usually squash all your commits in pull requests. The squashing approach tends to sum up to some rather large commits which in future debugging sessions are hard to work with. When there's too much noise it is hard to get to the bottom of a problem. Keeping commits as simple as possible makes maintaining the code easier in the future.

It is possible to do drive-by fixes in Cowboy-land without cluttering the commit log. Git has some wonderful tools that makes this an easy job.

Reply:
I agree that `git rebase --interactive` really *is* a marvellous tool for "telling the tale of what happened to the code" with much more hindsight based clarity than what would have been possible up front.

For maximum clarity, it does, however, require that I get used to doing *much* more fine grained commits than I currently tend to. I'm working on that under the brand name *commit early, commit often*, so for now, I am just abandoning this minor correction, and deferring that to some future PR of either "mostly cosmetic" changes or "more well structured bunches of rebased commits".

Reply:
I find `git commit --amend`  followed by `git push -f <remote> <branch>` very useful for  fixing minor issues when CI builds fails. That cuts down on most, if not all, of the small debugging commits.